staging/cse/windows/azurecnifunc.ps1

```diff
@@ -83,9 +83,11 @@ function Set-AzureCNIConfig
     } else {
         # Fill in DNS information for kubernetes.
         $exceptionAddresses = @()
-        if ($IsDualStackEnabled){
-            $subnetToPass = $KubeClusterCIDR -split ","
-            $exceptionAddresses += $subnetToPass[0]
+        if ($IsDualStackEnabled) {
+            $subnetsToPass = $KubeClusterCIDR -split ","

```

Comment:
will current rp pass multiple values in KubeClusterCIDR and this change will block anything?

Reply:
currently rp doesn't pass multiple values in kubeClusterCIDR. there is a separate PR for that. this shouldn't have any affect on any current clusters as IsDualStack shouldn't be enabled for any prod cluster.